testing/bdd/features/timing/computed_times.feature

```diff
@@ -5,8 +5,8 @@ Feature: Computed times computation
   # child elements.
 
   Examples:
-  | xml_file                             | sequence_identifier | sequence_number | no_body |  
-  | computed_resolved_time_semantics.xml | testSequence1       | 1               |         |  
+  | xml_file                             | sequence_identifier | sequence_number | no_body |
+  | computed_resolved_time_semantics.xml | testSequence1       | 1               |         |
 
   # SPEC-CONFORMANCE: R16 R17
```

Comment:
I wonder if we are missing a spec-conformance requirement to exclude elements from computed time calculation - this may have been missed. Ping @EyalLavi ?